src/components/InventoryHostStaleness/BaseDropDown.js

```diff
@@ -98,7 +99,7 @@ BaseDropdown.propTypes = {
   onSelect: PropTypes.bool,
   placeholder: PropTypes.string,
   title: PropTypes.string,
-  currentItem: PropTypes.string,
+  currentItem: PropTypes.any,
```

Comment:
🕵️‍♂️ Why this?

Reply:
newFormValues contains the 'id' string passed down from the api. So when we pass down newFormValues on page load, its an object of key/values of 6 numbers but one key === a string

Reply:
Then let's maybe use `PropTypes.oneOf([PropTypes.object, PropTypes.string])`. It is weird though that it changes type during runtime. If it is possible to correct that and make sure that currentItem is always either a string or an object, but doesn't change.

Reply:
It seems the decision to have this all be ints has been made on the backend, so ive changed this to number. I wont remove the string conversions until that backend peace is finished. That way if the review for this is completed beforehand, QE can test the full version, and if its not then I can just add on to this PR